libkbfs/md_ops.go

```diff
@@ -183,7 +191,14 @@ func (md *MDOpsStandard) processMetadata(ctx context.Context,
 		}
 	}
 
-	rmd := MakeRootMetadata(rmds.MD, extra, handle)
+	// TODO: Avoid the need for this copy (which is because MakeRootMetadata
```

Comment:
Is there a plan to avoid these deep copies?  Would be nice to have soon, even in this PR, if it's easy.